Sort the ecode list in the gnt-cluster manual for deterministic results.

Submitted upstream: <https://github.com/ganeti/ganeti/pull/1504>.

diff --git a/lib/build/sphinx_ext.py b/lib/build/sphinx_ext.py
--- a/lib/build/sphinx_ext.py
+++ b/lib/build/sphinx_ext.py
@@ -108,7 +108,7 @@ CV_ECODES_DOC = "ecodes"
 # pylint: disable=W0621
 CV_ECODES_DOC_LIST = [(name, doc) for (_, name, doc) in constants.CV_ALL_ECODES]
 DOCUMENTED_CONSTANTS = {
-  CV_ECODES_DOC: CV_ECODES_DOC_LIST,
+  CV_ECODES_DOC: sorted(CV_ECODES_DOC_LIST, key=lambda tup: tup[0]),
   }
 
 
